diff -up M2/configure.ac.mpir M2/configure.ac
--- M2/configure.ac.mpir	2013-05-21 09:48:43.018893373 -0500
+++ M2/configure.ac	2013-05-21 09:48:43.031893236 -0500
@@ -830,7 +830,7 @@ AC_SUBST(BUILTLIBS,)
 #    mpack depends on gmp (which mpir provides) mpfr mpc qd
 #    flint is experimental, depends on mpir and mpfr
 AC_SUBST(PROGLIST, "4ti2 gfan normaliz nauty cddplus lrslib")
-AC_SUBST(LIBLIST, "gc gdbm mpir mpfr pari readline ntl factory libfac lapack frobby glpk cddlib givaro fflas_ffpack linbox boost mpc qd mpack flint")
+AC_SUBST(LIBLIST, "gc gdbm gmp mpfr pari readline ntl factory libfac lapack frobby glpk cddlib givaro fflas_ffpack linbox boost mpc qd mpack flint")
 for i in $LIBLIST
 do eval BUILD_$i=no
 done
@@ -839,7 +839,7 @@ do eval BUILD_$i=yes
 done
 
 LIBLIST=" $LIBLIST "
-AC_ARG_ENABLE(build-libraries, AS_HELP_STRING(--enable-build-libraries=...,[list of libraries and programs to build from downloaded source code (e.g., gc gdbm mpir mpfr pari readline ntl gftables factory libfac lapack frobby glpk cddlib givaro fflas_ffpack linbox boost mpc qd mpack 4ti2 gfan normaliz nauty cddplus lrslib)]),
+AC_ARG_ENABLE(build-libraries, AS_HELP_STRING(--enable-build-libraries=...,[list of libraries and programs to build from downloaded source code (e.g., gc gdbm gmp mpfr pari readline ntl gftables factory libfac lapack frobby glpk cddlib givaro fflas_ffpack linbox boost mpc qd mpack 4ti2 gfan normaliz nauty cddplus lrslib)]),
     [for i in $enableval
     do case $LIBLIST in
 	    *" $i "*) eval BUILD_$i=yes ;;
@@ -946,16 +946,7 @@ dnl then BUILTLIBS="-lgmpxx -lgmp $BUILT
 dnl else LIBS="-lgmpxx $LIBS"
 dnl fi
 
-if test $BUILD_mpir = no
-then AC_LANG(C)
-     AC_SEARCH_LIBS(__mpir_version,mpir,,BUILD_mpir=yes)
-     AC_CHECK_HEADER(mpir.h,,BUILD_mpir=yes)
-fi
-dnl frobby uses mpirxx
-if test $BUILD_mpir = yes
-then BUILTLIBS="-lmpirxx -lmpir $BUILTLIBS"
-else LIBS="-lmpirxx $LIBS"
-fi
+LIBS="-lgmp -lgmpxx $LIBS"
 
 if test "$PARI" = yes
 then if test $BUILD_pari = no
