diff -p -up Macaulay2-1.4-r12617/configure.ac.orig Macaulay2-1.4-r12617/configure.ac
--- Macaulay2-1.4-r12617/configure.ac.orig	2010-11-10 21:42:52.682708000 -0200
+++ Macaulay2-1.4-r12617/configure.ac	2010-11-10 21:44:25.632708002 -0200
@@ -791,7 +791,7 @@ AC_SUBST(BUILTLIBS,)
 #	it writes a file into the user's home directory called ".polymake"
 #	it asks questions
 #    normaliz needs mpir providing libgmp and libgmpxx, but we provide that, anyway
-AC_SUBST(PROGLIST, "4ti2 gfan normaliz nauty cddplus lrslib")
+AC_SUBST(PROGLIST, "4ti2 gfan normaliz nauty lrslib")
 AC_SUBST(LIBLIST, "gc    gdbm    mpir    mpfr    pari    readline    ntl    factory        libfac         lapack    frobby    scscp	glpk    cddlib ")
 dnl I think we don't use this:
 dnl AC_SUBST(LIBFILES,"libgc libgdbm libmpir libmpfr libpari libreadline libntl libcf libcfmem libfac libblas liblapack libfrobby libscscp  libglpk ")
@@ -958,7 +958,7 @@ then AC_LANG(C)
      	[ 
 	  CPPFLAGS="-I/usr/include/cddlib $CPPFLAGS"
 	  unset ac_cv_header_cdd_h
-	  AC_CHECK_HEADER(cdd.h,,BUILD_cddlib=yes,[#include <setoper.h>])
+	  AC_CHECK_HEADER(cdd/cdd.h,,BUILD_cddlib=yes,[#include <cdd/setoper.h>])
 	],
 	[#include <setoper.h>])
 fi
