diff -up M2/configure.ac.no_nauty M2/configure.ac
--- M2/configure.ac.no_nauty	2013-03-16 04:32:37.000000000 -0500
+++ M2/configure.ac	2013-05-21 22:22:46.504602947 -0500
@@ -575,7 +575,6 @@ AC_SUBST(PACKAGES, dnl first Style, then
   Cyclotomic
   Binomials
   Kronecker
-  Nauty
   ToricVectorBundles
   ModuleDeformations
   PHCpack
@@ -593,7 +592,6 @@ AC_SUBST(PACKAGES, dnl first Style, then
   KustinMiller
   Units
   MonomialMultiplierIdeals
-  NautyGraphs
   VersalDeformations
   CharacteristicClasses
   RandomObjects
