diff -p -up Macaulay2-1.4-r12617/configure.ac.orig Macaulay2-1.4-r12617/configure.ac
--- Macaulay2-1.4-r12617/configure.ac.orig	2010-11-10 21:12:39.328708001 -0200
+++ Macaulay2-1.4-r12617/configure.ac	2010-11-10 21:12:48.000708001 -0200
@@ -992,33 +992,6 @@ then AC_LANG(C)
 fi
 test $BUILD_mpfr = yes && BUILTLIBS="-lmpfr $BUILTLIBS"
 
-if test $BUILD_gc = no
-then AC_LANG(C)
-     AC_SEARCH_LIBS(GC_free,gc,
-     	[AC_CHECK_HEADER(gc/gc.h,
-		[AC_CHECK_HEADER(atomic_ops.h,
-			[
-			  AC_MSG_CHECKING(whether libgc is recent enough)
-			  AC_RUN_IFELSE([ 
-				  #include <stdio.h>
-				  main () {
-				      extern unsigned GC_version;
-				      FILE *msg = fdopen(AS_MESSAGE_FD,"w");
-				      unsigned major, minor, alpha;
-				      major = GC_version >> 16;
-				      minor = (GC_version >> 8) & 0xff;
-				      alpha = GC_version & 0xff;
-				      if (alpha == 0xff) fprintf(msg,"(version %d.%d found) ", major, minor);
-				      else fprintf(msg,"(version %d.%d alpha %d found) ", major, minor, alpha);
-				      return !( major >= 7 ); }],
-			     [ AC_MSG_RESULT(yes) ],
-			     [ AC_MSG_RESULT([no, version at least 7.0 is required (gc will be built from downloaded sources)]); BUILD_gc=yes ],
-			     [ AC_MSG_RESULT([cross-compiling, test not possible]) ])
-			],
-			BUILD_gc=yes)],
-		BUILD_gc=yes)],
-	BUILD_gc=yes)
-fi
 AC_SUBST(GC_LIBS,-lgc)
 if test $BUILD_gc = yes
 then if test $PTHREADS = yes
diff -p -up Macaulay2-1.4-r12617/Macaulay2/d/version.dd.orig Macaulay2-1.4-r12617/Macaulay2/d/version.dd
--- Macaulay2-1.4-r12617/Macaulay2/d/version.dd.orig	2010-11-11 16:16:07.845708117 -0200
+++ Macaulay2-1.4-r12617/Macaulay2/d/version.dd	2010-11-11 16:16:33.454708019 -0200
@@ -62,7 +62,7 @@ header "
    ";
 gcversion():constcharstar := Ccode(returns,"
      static char buf[40];
-     unsigned x = GC_get_version();
+     unsigned x = GC_version;
      unsigned major = x >> 16;
      unsigned minor = (x >> 8) & 0xff;
      unsigned alpha = x & 0xff;
diff -p -up Macaulay2-1.4-r12617/Macaulay2/d/actors5.d.orig Macaulay2-1.4-r12617/Macaulay2/d/actors5.d
--- Macaulay2-1.4-r12617/Macaulay2/d/actors5.d.orig	2010-11-11 16:40:05.335708004 -0200
+++ Macaulay2-1.4-r12617/Macaulay2/d/actors5.d	2010-11-11 17:40:22.075708316 -0200
@@ -1937,16 +1937,16 @@ GCstats(e:Expr):Expr := (
      if length(s) == 0 then Expr(toHashTable(Sequence(
 		    "heap size" => toExpr(Ccode(int,"GC_get_heap_size()")),
 		    "free space divisor" => toExpr(Ccode(int,"GC_free_space_divisor")),
-		    "number of collections" => toExpr(Ccode(int,"GC_get_gc_no()")),
-		    "parallel" => toExpr(Ccode(bool,"!!GC_get_parallel()")),
-		    "all interior pointers" => toExpr(Ccode(bool,"GC_get_all_interior_pointers()")),
-		    "finalize on demand" => toExpr(Ccode(bool,"!!GC_get_finalize_on_demand()")),
-		    "java finalization" => toExpr(Ccode(bool,"GC_get_java_finalization()")),
-		    "don't expand" => toExpr(Ccode(bool,"GC_get_dont_expand()")),
+		    "number of collections" => toExpr(Ccode(int,"GC_gc_no")),
+		    "parallel" => toExpr(Ccode(bool,"!!GC_parallel")),
+		    "all interior pointers" => toExpr(Ccode(bool,"GC_all_interior_pointers")),
+		    "finalize on demand" => toExpr(Ccode(bool,"!!GC_finalize_on_demand")),
+		    "java finalization" => toExpr(Ccode(bool,"GC_java_finalization")),
+		    "don't expand" => toExpr(Ccode(bool,"GC_dont_expand")),
 		    "use entire heap" => toExpr(Ccode(bool,"GC_use_entire_heap")),
-		    "full freq" => toExpr(Ccode(int,"GC_get_full_freq()")),
-		    "max retries" => toExpr(Ccode(int,"GC_get_max_retries()")),
-		    "time limit" => toExpr(Ccode(ulong,"GC_get_time_limit()")),
+		    "full freq" => toExpr(Ccode(int,"GC_full_freq")),
+		    "max retries" => toExpr(Ccode(int,"GC_max_retries")),
+		    "time limit" => toExpr(Ccode(ulong,"GC_time_limit")),
 		    "GC_INITIAL_HEAP_SIZE" => toExpr(getenv("GC_INITIAL_HEAP_SIZE")),
 		    "GC_MAXIMUM_HEAP_SIZE" => toExpr(getenv("GC_MAXIMUM_HEAP_SIZE")),
 		    "GC_LOOP_ON_ABORT" => toExpr(getenv("GC_LOOP_ON_ABORT")),
diff -p -up Macaulay2-1.4-r12617/Macaulay2/bin/Makefile.in.orig Macaulay2-1.4-r12617/Macaulay2/bin/Makefile.in
--- Macaulay2-1.4-r12617/Macaulay2/bin/Makefile.in.orig	2010-11-11 18:35:27.975708062 -0200
+++ Macaulay2-1.4-r12617/Macaulay2/bin/Makefile.in	2010-11-11 18:35:49.085707753 -0200
@@ -111,7 +111,7 @@ else
 MAYBESTATIC = $(UNSTATIC)
 endif
 
-M2_LIBRARIES :=
+M2_LIBRARIES := @GC_LIBS@
 
 ifeq (@STATIC@,yes)
 M2_LIBRARIES += $(MAYBESTATIC)
